src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -404,6 +427,122 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 11,
                 "List of registered output formatters, in the following order: {OutputFormatters}");
+
+            _registeredModelBinderProviders = LoggerMessage.Define<IModelBinderProvider[]>(
+                LogLevel.Debug,
+                12,
+                "Registered model binder providers, in the following order: {ModelBinderProviders}");
+
+            _attemptingToBindPropertyModel = LoggerMessage.Define<string, Type, Type, string>(
+               LogLevel.Debug,
+               13,
+               "Attempting to bind property '{PropertyName}'[{ModelType}] of type '{PropertyContainerType}' using the name '{ModelName}' in request data ...");
```

Comment:
`'MyProperty'[System.String] of type 'Application.Models.MyModel'` is very confusing even if you stop using "of type" backwards (it's "in type" here). Not a fan of the square brackets anywhere in the new logging.

Suggest `'{PropertyContainerType}.{PropertyName}' of type '{ModelType}'` e.g `'Application.Models.MyModel.MyProperty`' of type 'System.String'` everywhere.